docker-compose.yml

```diff
@@ -65,18 +73,6 @@ services:
       file: hyrax-webapp/docker-compose.yml
       service: db
 
-  adminer:
-    extends:
-      file: hyrax-webapp/docker-compose.yml
-      service: adminer
-
-  base:
```

Comment:
as part of the experiment I removed the base build since web builds base anyways -- this was to help cut down on build time and only removed as a test.